web/src/main/java/com/epam/rft/atsy/web/validator/FileValidatorImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.rft.atsy.web.validator;
+
+
+import com.epam.rft.atsy.service.exception.file.FileIsEmptyValidationException;
+import com.epam.rft.atsy.service.exception.file.FileIsInWrongSyntaxValidationException;
+import com.epam.rft.atsy.service.exception.file.FileIsMissingValidationException;
+import com.epam.rft.atsy.service.exception.file.FileIsTooLargeValidationException;
+import com.epam.rft.atsy.service.exception.file.FileValidationException;
+
+import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Component
+public class FileValidatorImpl implements FileValidator {
+
+  private static final long MAX_FILE_SIZE = 1024 * 1024 * 10; // 10 MB
+  // Example: Garfield_Hungry-Angry_2010-12-24.doc
+  private static final String FILE_PATTERN = "([A-Z][a-z]+[-|_])+[0-9-]+[.](doc|docx|odt|pdf)$";
```

Comment:
I think this check should be separated. We should have a different message if the extension is not supported, and a different one if the filename (the part before the extension) contains invalid characters.